cumulusci/robotframework/pageobjects/PageObjects.py

```diff
@@ -79,11 +80,16 @@ def __init__(self, *args):
         importer = robot.utils.Importer()
 
         for file_path in args:
-            try:
-                importer.import_class_or_module_by_path(os.path.abspath(file_path))
-                logger.debug("imported page object {}".format(file_path))
-            except Exception as e:
-                logger.warn(str(e))
+            path = self._find_file_in_pythonpath(file_path)
+            if path:
+                try:
+                    importer.import_class_or_module_by_path(os.path.abspath(path))
+                    logger.debug(f"imported page object {path}")
+                except Exception as e:
+                    logger.warn(str(e))
```

Comment:
```suggestion
                    logger.warning(str(e))
```

(`warn` is deprecated)

Also, why are we using a warning here instead of letting the exception be raised? Doesn't that just hide the fact that the intended page object couldn't be imported, which could lead to confusing errors if something tries to use that page object later?

Reply:
That's a good question.  I don't remember why I thought a warning was more appropriate. Now that you bring it up, it does seem like this ought to throw an error. I've changed it to throw errors, and I've written a couple of tests.